editoast/openapi.yaml

```diff
@@ -858,3 +831,38 @@ components:
             is_warning: false
             obj_id: 61205924-6667-11e3-81ff-01f464e0362d
             obj_type: TrackSection
+
+    ComputeTrackRangesErrors:
+      NotFound:
```

Comment:
such nesting is not valid openapi, please validate the file with https://editor.swagger.io/

Reply:
done !